workflows/files/ZWEWRF05.xml

```diff
@@ -174,12 +174,12 @@ and that will be also used to secure newly generated keystores for API Mediation
         <variableValue name="ZOWE_GROUP_ID" scope="instance" noPromptIfSet="false" required="true"/>
 		<instructions substitution="false">Run this step to initialize variable values</instructions>
 		<weight>1</weight>
-		<autoEnable>false</autoEnable>
+		<autoEnable>true</autoEnable>
```

Comment:
We wanted to have the possibility to run all workflows automatically and this step was the last one that does not have all step auto enabled. So I added this change to this pull request.